bower.json

```diff
@@ -30,7 +30,8 @@
     "purescript-maybe": "^2.0.1",
     "purescript-unsafe-coerce": "^2.0.0",
     "purescript-bifunctors": "^2.0.0",
-    "purescript-refs": "^2.0.0"
+    "purescript-refs": "^2.0.0",
+    "purescript-foreign": ">=3.0.1"
```

Comment:
Just using an `^` constraint here should still work to set a minimum bound - this is probably fine too, but it'll match v4, v5, etc. too, which we probably don't want?